Privacy.md

```diff
@@ -0,0 +1,88 @@
+# Forecastie
+
+# Privacy Policy
+
+This privacy policy statement is created to clearly state the developer's policies with respect to data collection, usage, and security. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized, and the choices you have associated with that data.
```

Comment:
I care more about the software in question than what policies the developer has.